### PR TITLE
0.21.0 release prep

### PR DESCRIPTION
**Pull request by rlankfo**
Prepare for v0.21.0 release by following the [release procedure](https://github.com/grafana/agent/blob/main/cmd/agent/DEVELOPERS.md#performing-the-release)

### CODE REVIEW

**Comment by rfratto** [comment]
Your branch is three commits behind main, can you fast forward?

File: pkg/operator/defaults.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ var (
 		"v0.18.3",
 		"v0.18.4",
 		"v0.19.0",
-		"v0.20.0",
+		"v0.21.0",
```
Comment (rfratto) [inline pkg/operator/defaults.go:17]: This should add a new version instead of changing the old one

**Review by rfratto — COMMENTED** [review]
LGTM aside from two comments

File: pkg/operator/defaults.go (hunk 1 above)
Comment (rlankfo) [reply pkg/operator/defaults.go:18]: woops, did some find / sed here and missed excluded this one.

**Comment by rlankfo** [comment]
> Your branch is three commits behind main, can you fast forward?

fixed. did not fetch upstream before creating my branch locally.

**Review by rfratto — APPROVED** [review]
LGTM, though now that you pulled there's one more v0.20.0 to update :)

**Comment by rlankfo** [comment]
> LGTM, though now that you pulled there's one more v0.20.0 to update :)

got it 😅

**Review by mattdurham — APPROVED** [review]
Lgtm assuming we don’t sneak anymore changes in :)

**Pull request merged**